groups/script.js

```diff
@@ -61,16 +59,15 @@ const handler = {
         renderAllGroups({
           cardOnClick: groupCardOnAction,
         });
-        if (isDev && (!value || value.length == 0)) renderNoGroupFound();
+        if (!value || value.length == 0) renderNoGroupFound();
         break;
       case 'search':
-        if (isDev) {
-          setParamValueInURL(QUERY_PARAM_KEY.GROUP_SEARCH, value);
-          dataStore.filteredGroupsIds = getDiscordGroupIdsFromSearch(
-            Object.values(dataStore.groups),
-            value,
-          );
-        } else if (value === '') {
+        setParamValueInURL(QUERY_PARAM_KEY.GROUP_SEARCH, value);
+        dataStore.filteredGroupsIds = getDiscordGroupIdsFromSearch(
+          Object.values(dataStore.groups),
+          value,
+        );
+        if (value === '') {
```

Comment:
This code is not required since `getDiscordGroupIdsFromSearch` function do the same
```
        if (value === '') {
          if (dataStore.groups == null) break;
          dataStore.filteredGroupsIds = Object.values(dataStore.groups).map(
            (group) => group.id,
          );
        } else {
          const search = value.toLowerCase();
          dataStore.filteredGroupsIds = Object.values(dataStore.groups)
            .filter((group) => group.title.toLowerCase().includes(search))
            .map((group) => group.id);
        }

Reply:
made the changes please check once

Reply:
Thank you